### DISCUSSION TITLE
tutorials/intro-typescript/01-course-overview-and-setup

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#139] [course feedback]
# tutorials/intro-typescript/01-course-overview-and-setup

Learn about the GraphQL API we'll build in this series

https://www.apollographql.com/tutorials/intro-typescript/01-course-overview-and-setup

**Comment by zerone-jang** [comment]
nice and neat explanation

**Comment by kristiyan-velkov** [comment]
The explanation of GraphQL in this tutorial is excellent. It breaks down complex concepts into easily understandable segments, making it a valuable resource for both beginners and experienced developers.

If you want to read more about similar topics, check out my [Medium blog](https://medium.com/@kristiyan.velkov) where I post regularly. 

Your feedback and engagement are always welcome!

**Comment by dekelly** [comment]
Good introduction and all steps completed with no issues.

**Comment by kelvenus** [comment]
I am new to GraphQL, would love to see what I can build. First, I will be trail to finish this lab.

**Comment by Devansh0012** [comment]
Thought it would be tough to learn, but really great explanation!

**Comment by abid365** [comment]
This tutorial is well documented and maintained. Kudos to the maintainers! 🎊
